setup_bitswap.go

```diff
@@ -21,7 +23,16 @@ import (
 
 func setupBitswapExchange(ctx context.Context, cfg Config, h host.Host, cr routing.ContentRouting, bstore blockstore.Blockstore) exchange.Interface {
 	bsctx := metri.CtxScope(ctx, "ipfs_bitswap")
-	bn := bsnet.NewFromIpfsHost(h, cr)
+
+	bn := bsnet.NewFromIpfsHost(h)
+
+	// Custom query manager with the content router and the host
+	// and our custom options to overwrite the default.
+	pqm, err := providerquerymanager.New(ctx, h, cr, providerquerymanager.WithMaxInProcessRequests(100))
```

Comment:
Note: This can also be set to 0 for unlimited. Would it be safer to use `-1` for unlimited and treat `0` as requesting the default?

Reply:
Since this is an option, using the option is what changes the default. It is like that for most options at least, so I'm against attaching more semantics.

Reply:
That said we can test with 0 too